[2025-04-17T07:45:30.205+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-17T07:45:30.224+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data scheduled__2025-04-13T07:58:00+00:00 [queued]>
[2025-04-17T07:45:30.233+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data scheduled__2025-04-13T07:58:00+00:00 [queued]>
[2025-04-17T07:45:30.234+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2025-04-17T07:45:30.252+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-13 07:58:00+00:00
[2025-04-17T07:45:30.258+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=459) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-17T07:45:30.260+0000] {standard_task_runner.py:72} INFO - Started process 463 to run task
[2025-04-17T07:45:30.260+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'scheduled__2025-04-13T07:58:00+00:00', '--job-id', '246', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmpvmg6m214']
[2025-04-17T07:45:30.261+0000] {standard_task_runner.py:105} INFO - Job 246: Subtask extract_raw_data
[2025-04-17T07:45:30.317+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data scheduled__2025-04-13T07:58:00+00:00 [running]> on host c0ac39583bd8
[2025-04-17T07:45:30.411+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T07:58:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T07:58:00+00:00'
[2025-04-17T07:45:30.413+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-17T07:45:30.428+0000] {lead_data_pipeline.py:146} INFO - Started batch processing for 2025-04-17.
[2025-04-17T07:45:30.429+0000] {lead_data_pipeline.py:35} INFO - Created synthetic data for RO country code.
[2025-04-17T07:45:30.647+0000] {lead_data_pipeline.py:103} INFO - Written 414 records to the CSV file.
[2025-04-17T07:45:31.923+0000] {lead_data_pipeline.py:119} INFO - Added UUID to the dataset.
[2025-04-17T07:45:32.265+0000] {lead_data_pipeline.py:138} INFO - Updated accessed timestamp.
[2025-04-17T07:45:32.267+0000] {lead_data_pipeline.py:154} INFO - Finished batch processing 2025-04-17.
[2025-04-17T07:45:32.268+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-17T07:45:32.280+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-17T07:45:32.281+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=scheduled__2025-04-13T07:58:00+00:00, execution_date=20250413T075800, start_date=20250417T074530, end_date=20250417T074532
[2025-04-17T07:45:32.333+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-17T07:45:32.363+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-17T07:45:32.370+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
